docs/get-started.md

```diff
@@ -275,7 +275,7 @@ There are three kinds of tags (_course_, _building_ and _team_) and two edge typ
       "select":[  
          "grade: integer"
       ],
-      "like":[  
+      like":[  
```

Comment:
why remove the `quotation`? 

Please see [Here](https://github.com/vesoft-inc/nebula/blob/72635a2f2888bbf64cefd68537e754e534142ebc/docs/get-started.md)  It's looks like failed.

Reply:
Fixed, it was an accident.